src/dfi/consensus/governance.cpp

```diff
@@ -138,6 +176,37 @@ Res CGovernanceConsensus::operator()(const CGovernanceUnsetMessage &obj) const {
         if (!res) {
             return Res::Err("%s: %s", name, res.msg);
         }
+    }
+
+    // Store pending Gov var changes
+    return StoreUnsetGovVars(obj, mnview);
+}
+
+Res CGovernanceConsensus::operator()(const CGovernanceUnsetMessage &obj) const {
```

Comment:
should have a DF24 check to no longer allow it afterwards, right?

Reply:
Same as the other. So, this should be ok

https://github.com/DeFiCh/ain/pull/3063/files#diff-2e2980eda773148aea9f732c9f165bb7a9427b4f9df3c49b78b16d9166d58c42R289-R293